src/aics-image-viewer/components/App/index.tsx

```diff
@@ -1077,6 +1091,7 @@ export default class App extends React.Component<AppProps, AppState> {
             mode={userSelections[MODE]}
             imageType={userSelections.imageType}
             autorotate={userSelections[AUTO_ROTATE]}
+            showAxes={userSelections.showAxes}
```

Comment:
for naming consistency, maybe this should be `userSelections[SHOW_AXES]` ?

Reply:
We do both in this block. The convention of using constants like this is a hold over from Javascript I think. With typescript you get better warnings and autocomplete using the dot notation

Reply:
There is nonetheless a `SHOW_AXES` constant used everywhere but here, so for consistency with the rest of this change this looks like a good catch to me.

Reply:
I agree with Megan that we could start removing those string constants as we go, in favor of explicit names knowing that we have typing.